<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>virtual_hdd_ledConfig</class>
 <widget class="QWidget" name="virtual_hdd_ledConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>334</width>
    <height>171</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Configure virtual hdd led</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Poll interval (ms):</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSpinBox" name="PollIntervalspinBox">
       <property name="toolTip">
        <string>intervall for polling the data for the disk (milliseconds) </string>
       </property>
       <property name="minimum">
        <number>100</number>
       </property>
       <property name="maximum">
        <number>1000</number>
       </property>
       <property name="value">
        <number>250</number>
       </property>
      </widget>
     </item>
     <item row="0" column="2" colspan="2">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>47</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Disk name:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="KComboBox" name="disknamekcombobox">
       <property name="toolTip">
        <string>name of the disk which should be monitored e.g. sda, hda</string>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
       <item>
        <property name="text">
         <string>sda</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>sdb</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>hda</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>sdc</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>sdd</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>sde</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>sdf</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>sdg</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>sdh</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>hdb</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>hdc</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>hdd</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="2" colspan="2">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>168</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QCheckBox" name="draw_devicenamecheckBox">
       <property name="toolTip">
        <string>draw the device name below the led</string>
       </property>
       <property name="text">
        <string> draw devicename too</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2" colspan="2">
      <widget class="QCheckBox" name="draw_bordercheckBox">
       <property name="toolTip">
        <string>draw a black border around the led</string>
       </property>
       <property name="text">
        <string> draw border around led</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Read color:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="KColorButton" name="Readkcolorbutton">
       <property name="toolTip">
        <string>color if led state is read</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Write color:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="KColorButton" name="Writekcolorbutton">
       <property name="toolTip">
        <string>color if led state is write</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Read/Write color:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="KColorButton" name="ReadWritekcolorbutton">
       <property name="toolTip">
        <string>color if led state is read and write</string>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Idle color:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="3">
      <widget class="KColorButton" name="Idlekcolorbutton">
       <property name="toolTip">
        <string>color if led state is idle (no read and no write)</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Unknown color:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="KColorButton" name="Unknownkcolorbutton">
       <property name="toolTip">
        <string>color if led state is unknown or disk not found</string>
       </property>
      </widget>
     </item>
     <item row="6" column="2">
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>58</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="6" column="3">
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>68</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="3" column="0" colspan="4">
      <widget class="QCheckBox" name="use_iconscheckBox">
       <property name="toolTip">
        <string>use nice ions instead plain color (slower)</string>
       </property>
       <property name="text">
        <string>use icons instead of plain color</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <spacer name="horizontalSpacer_4">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>15</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>9</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KColorButton</class>
   <extends>QPushButton</extends>
   <header>kcolorbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
